### PR TITLE
Merge image plugins

### PR DESCRIPTION
**Pull request by luukvbaal**
Move `lsix` from `img/vidthumb` to `imgview`. Also generate audio thumbnails.
Rewrote `imgview` how I thought best to maintain the functionality of all 3 plugins, please try it out.

### CODE REVIEW

File: plugins/imgview [hunk 1]
```diff
@@ -1,72 +1,101 @@
...
+#              and deleted on exit. Supported image viewers open
+#              in thumbnail mode when the hovered entry is a directory.
 #
 # Dependencies:
 #   - imv (https://github.com/eXeC64/imv) or,
 #   - sxiv (https://github.com/muennich/sxiv) or,
-#   - viu (https://github.com/atanunq/viu), less
+#   - viu (https://github.com/atanunq/viu), or
+#   - catimg (https://github.com/posva/catimg), or
+#   - lsix (https://github.com/hackerb9/lsix)
```
Comment (jarun) [inline plugins/imgview:13]: Please add ffmpeg(thumbnailer) as a deps.

Check out https://github.com/ckardaris/ucollage as well. Leaving ti up to you to add it or not.

**Review by jarun — CHANGES_REQUESTED** [review]

**Comment by jarun** [comment]
Thank you!

**Comment by luukvbaal** [comment]
Was still working on your requested changes and checking out `ucollage`😅. I think it can be included.

**Comment by Kabouik** [comment]
I found out about `ucollage` too on Reddit recently and really liked it!

**Comment by jarun** [comment]
Just raise another PR. No issues!

**Pull request merged**